src/my-workspace/views/Bookmarks.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { Fragment, FunctionComponent } from 'react';
+
+import {} from '@viaa/avo2-components';
+import { Avo } from '@viaa/avo2-types';
+import { RouteComponentProps } from 'react-router';
+
+interface BookmarksProps extends RouteComponentProps {}
+
+export const Bookmarks: FunctionComponent<BookmarksProps> = ({  }: BookmarksProps) => {
```

Comment:
Je moet die `: BookmarkProps` niet meer specifiek definiëren in uw function params aangezien dat hier `: FunctionComponent<BookmarksProps>` al gebeurd.